src/main/java/net/neoforged/moddevgradle/tasks/JarJar.java

```diff
@@ -36,7 +36,7 @@ public abstract class JarJar extends DefaultTask {
     @Inject
     public JarJar(FileSystemOperations fileSystemOperations) {
         this.fileSystemOperations = fileSystemOperations;
-        this.getOutputDirectory().convention(getProject().getLayout().getBuildDirectory().dir("jarJar/"+getName()));
+        this.getOutputDirectory().convention(getProject().getLayout().getBuildDirectory().dir("generated/jarjar/" + getName()));
```

Comment:
I'm not so sure about `generated/jarJar/taskName`. Can it not just be `generated/taskName`? Or `build/taskName`.